src/skill_tree.md

```diff
@@ -72,22 +78,46 @@ href = "https://github.com/rust-lang/rust/issues/53020"
 requires = []
 items = []
 
+[[group]]
+name = "unsafe_rules"
+label = "unsafe operations in CTFE"
+href = "https://github.com/rust-lang/rfcs/pull/3016"
+items = []
+
 [[group]]
 name = "unconst_rules"
-label = "Need to come up\nwith a scheme\nfor const unsafe/unconst"
-items = [
-  { label = "feature:const_fn_transmute", href = "https://github.com/rust-lang/rust/issues/53605" },
-  { label = "feature:const_fn_union", href = "https://github.com/rust-lang/rust/issues/51909", port = "union" },
-  { label = "feature:const_raw_ptr_deref", href = "https://github.com/rust-lang/rust/issues/51911", port = "raw_ptr_deref" },
-]
+label = "unconst operations"
 href = "https://github.com/rust-lang/const-eval/issues/14"
+items = []
+requires = ["unsafe_rules"]
+
+[[group]]
+name = "transmute"
+label = "feature:const_fn_transmute"
+items = []
+requires = ["unconst_rules"]
```

Comment:
Why does "transmute" require "unconst" when the very similar union/raw-ptr require "unsafe"?

Reply:
hmm overlooked that one

Reply:
force pushed the last commit